renovate.json

```diff
@@ -65,7 +65,8 @@
     {
       "groupName": "@types/node",
       "matchPackageNames": ["@types/node"],
-      "allowedVersions": "< 21.0.0"
+      "allowedVersions": "< 21.0.0",
+      "automerge": true
```

Comment:
This is a `devDep`, it should be automerging. Could it be that the `"resolutions"` is blocking it?

Reply:
Yes, I think it is the root cause of it not being merged. 🙈

Reply:
And—it worked. 👍 
https://github.com/mui/mui-x/pull/15579#event-15419927633